api/base/permissions.py

```diff
@@ -53,6 +54,12 @@ def _get_scopes(self, request, view):
             except AttributeError:
                 raise ImproperlyConfigured('TokenHasScope requires the view to define the '
                                            'required_read_scopes attribute')
+            assert is_iterable_but_not_string(view.required_read_scopes), \
+                'The required_read_scopes must be an iterable of CoreScopes'
+            if view.required_read_scopes and isinstance(view.required_read_scopes[0], tuple):
```

Comment:
> @brianjgeiger, the view.required_read_scopes will always contain CoreScopes from what I've seen. 

And correct: each item in `required_read_scopes` should be a value (which for argument's sake is usually a string), and that value should be attached as one of the class-level attributes of `CoreScopes`. Eg:

`required_read_scopes = [CoreScopes.INSTITUTION_READ, CoreScopes.USERS_READ]`

Where `INSTITUTION_READ = 'institutions_read'`